e2e_deployment_files/connectivity_deployment.template.json

```diff
@@ -596,6 +626,15 @@
           "timeToLiveSecs": 86400
         }
       }
+    },
+    "metricsCollector": {
+      "properties.desired": {
+        "upstreamProtocol": "<UpstreamProtocol>",
+        "offlineFrequency": "<NetworkController.OfflineFrequency0>",
+        "onlineFrequency": "<NetworkController.OnlineFrequency0>",
+        "runsCount": "<NetworkController.RunsCount0>",
```

Comment:
please add prefix "networkController" for offlineFrequency,  onlineFrequency and runsCount.